application/config/config.go

```diff
@@ -510,10 +510,20 @@ func (c *Config) ConfigureLogging(server lsp.Server) {
 		w.Out = multiLevelWriter
 		w.NoColor = true
 		w.TimeFormat = time.RFC3339
+		w.PartsOrder = []string{
+			zerolog.TimestampFieldName,
+			zerolog.LevelFieldName,
+			"method",
+			"ext",
+			"separator",
+			zerolog.CallerFieldName,
+			zerolog.MessageFieldName,
+		}
+		w.FieldsExclude = []string{"method", "separator", "ext"}
```

Comment:
What does this do? Is the method field still displayed in debug logs? If not, I'm against excluding it.

Reply:
Sorry I will provide an example of the log outputs in the description, but what it needs to be seen with the configuration above. The fields default representation is suppressed, while they are intentionally included on top without the key=value representation.